SQL/New_patches/2020_06_16_Add_Date_Format_to_ConfigSettings_DataType.sql

```diff
@@ -0,0 +1,11 @@
+-- Add date_format option to DataType
```

Comment:
I wouldnt mind seeing in this patch. conversion from any casing  combination into the supported format

ie

`UPDATE Config SET Value='Ymd' WHERE Value IN ('YMD','YMd','ymd',...) AND ConfigID=(SELECT ID ...);`

and a similar update for the different combinations of YM

Reply:
@ridz1208 implemented.